main.go

```diff
@@ -79,8 +79,13 @@ func realMain(ctx context.Context) error {
 	if *flagBind == "" {
 		return fmt.Errorf("missing -bind")
 	}
-	if *flagServerUpMinutes < 0 {
-		return fmt.Errorf("-server-up-minutes should not be negative")
+	var d time.Duration
+	var err error
```

Comment:
Nit: Move this inside the conditional, since otherwise it leaks the scope.

Reply:
done